src/main/resources/application.properties

```diff
@@ -10,10 +10,16 @@ spring.jpa.hibernate.ddl-auto=none
 # FLYWAY
 spring.flyway.baseline-on-migrate=true
 spring.flyway.enabled=true
+spring.flyway.clean-disabled=false
 spring.flyway.locations=classpath:db/migration
 
+# LOGGING
```

Comment:
`INFO` is a default level I guess? Do you need to set it explicitly?

Reply:
you are totally right.

Reply:
So why are you changing those?

Reply:
not knowing their exact functioning, I changed them, to try out different levels of logging.
now I removed the INFO levels